src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -127,15 +137,16 @@ export class MoreInfoDialog extends LitElement {
                 `
               : ""}
           </ha-header-bar>
-          ${this._computeShowHistoryComponent(entityId)
+          ${CONTROL_DOMAINS.includes(domain) &&
+          this._computeShowHistoryComponent(entityId)
             ? html`
                 <mwc-tab-bar
                   .activeIndex=${this._currTabIndex}
                   @MDCTabBar:activated=${this._handleTabChanged}
                 >
                   <mwc-tab
                     .label=${this.hass.localize(
-                      "ui.dialogs.more_info_control.controls"
+                      "ui.dialogs.more_info_control.details"
```

Comment:
Its not just details though... But I guess this works :)

Reply:
This is fine until we change the names of entities 😉 then I think we should change it to match what they are.

Reply:
Or we could make it Entity for now until we change the naming